packages/contracts-periphery/contracts/foundry-tests/OptimistInviter.t.sol

```diff
@@ -0,0 +1,689 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+/* Testing utilities */
+import { Test } from "forge-std/Test.sol";
+import { AttestationStation } from "../universal/op-nft/AttestationStation.sol";
+import { OptimistInviter } from "../universal/op-nft/OptimistInviter.sol";
+import { Optimist } from "../universal/op-nft/Optimist.sol";
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+import { TestERC1271Wallet } from "../testing/helpers/TestERC1271Wallet.sol";
+
+contract OptimistInviter_Initializer is Test {
+    event InviteClaimed(address indexed issuer, address indexed claimer);
+    event Initialized(uint8 version);
+    event Transfer(address indexed from, address indexed to, uint256 indexed tokenId);
+    event AttestationCreated(
+        address indexed creator,
+        address indexed about,
+        bytes32 indexed key,
+        bytes val
+    );
+
+    bytes32 CLAIMABLE_INVITE_TYPEHASH;
+    bytes32 EIP712_DOMAIN_TYPEHASH;
+
+    address constant alice_inviteGranter = address(128);
+    address constant sally = address(512);
+    address constant ted = address(1024);
+    address constant eve = address(2048);
+
+    address internal bob;
+    uint256 internal bobPrivateKey;
+    address internal carol;
+    uint256 internal carolPrivateKey;
+
+    uint256 currentNonce;
+
+    TestERC1271Wallet carolERC1271Wallet;
+
+    AttestationStation attestationStation;
+    OptimistInviter optimistInviter;
+
+    function setUp() public {
+        currentNonce = 0;
+
+        bobPrivateKey = 0xB0B0B0B0;
+        bob = vm.addr(bobPrivateKey);
+
+        carolPrivateKey = 0xC0C0C0C0;
+        carol = vm.addr(carolPrivateKey);
+
+        carolERC1271Wallet = new TestERC1271Wallet(carol);
+
+        // Give alice and bob and sally some ETH
+        vm.deal(alice_inviteGranter, 1 ether);
+        vm.deal(bob, 1 ether);
+        vm.deal(sally, 1 ether);
+        vm.deal(eve, 1 ether);
+
+        vm.label(alice_inviteGranter, "alice_inviteGranter");
```

Comment:
there is a helper function in forge-std `makeAddr` that will create an address and call `vm.label`, could be helpful to begin migrating to that